app/views/templates/report-details.html.slim

```diff
@@ -5,7 +5,9 @@
   .project-version-labels.clearfix
     .label.label-primary ng-if='!report' Loading...
     span ng-repeat='project in report.projects'
-      span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}' {{ project | projectName }} {{ version.name }}
+      span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}'
+        a.label-link href='#' ui-sref='org.projects.show({ orgName: organization.name, projectName: project.name })' {{ project | projectName }}
+        |  {{ version.name }}
```

Comment:
I would put the link around the version name as well. We might later link to the project page with some version-specific widgets pre-configured. But for the foreseeable future, we're not going to link to a page showing only information about the version and not the projet.